charts/camunda-platform/templates/tasklist/deployment.yaml

```diff
@@ -75,6 +75,13 @@ spec:
               {{- end }}
             - name: CAMUNDA_TASKLIST_IDENTITY_AUDIENCE
               value: "tasklist-api"
+            {{- if .Values.tasklist.identityUserAccessRestrictionsEnabled }}
```

Comment:
Another point: if that's a bool, there's no need to use the if condition just use it directly in the var.

Reply:
But if it is not set, the default is true, do I need any special part for this? or its fine my default definition on my app?

Reply:
If you want it enabled by default, then the new implementation is what we are looking for :ok_hand: